includes/ECommerce.php

```diff
@@ -624,14 +624,14 @@ public function custom_status_column_content( $column_name, $post_id ) {
 				$label_text       = __( 'Published - Private', 'wp-module-ecommerce' );
 			} else {
 				$background_color = '#E8ECF0';
-				$label_text       = __( $post_status, 'wp-module-ecommerce' );
+				$label_text       = $post_status;
```

Comment:
Since adding translation cannot be done dynamically, removing it from $post_status.